crates/agglayer-node/src/rpc/tests/send_certificate.rs

```diff
@@ -40,7 +41,17 @@ async fn send_certificate_method_can_be_called_and_succeed() {
         config.clone(),
     );
 
-    let _server_handle = AgglayerImpl::new(Arc::new(service)).start().await.unwrap();
+    let router = AgglayerImpl::new(Arc::new(service)).start().await.unwrap();
+
+    let listener = tokio::net::TcpListener::bind(config.rpc_addr())
+        .await
+        .unwrap();
+    let api_server = axum::serve(listener, router);
+
+    let _rpc_handle = tokio::spawn(async move {
+        _ = api_server.await;
```

Comment:
What does `api_server.await` return? Would it make sense to log it?

Reply:
`axum::serve` returns a `anyhow::Result<()>`, but as we're in a test file I think we're good